pkg/api/v1alpha1/cluster_types.go

```diff
@@ -239,6 +240,7 @@ type WorkerNodeGroupConfiguration struct {
 	// Taints define the set of taints to be applied on worker nodes
 	Taints []corev1.Taint `json:"taints,omitempty"`
 	// Labels define the labels to assign to the node
+	// NOTE: CAPC provider supports using 'ds.meta_data.failuredomain' as label value and replaces it with proper failuredomain value.
```

Comment:
Same as above.

Reply:
Since eks-a doc is being worked on by some other resource, create an issue and notify :
https://github.com/aws/eks-anywhere/issues/3735